application/back-office/WebApp/package.json

```diff
@@ -6,26 +6,13 @@
     "dev": "npm run update-translations && npm run swagger && rsbuild dev",
     "build": "npm run update-translations && npm run swagger && rsbuild build",
     "update-translations": "lingui extract && lingui compile --typescript",
-    "check": "biome check",
-    "lint": "biome lint --write",
-    "format": "biome format --write",
     "swagger": "openapi-typescript shared/lib/api/BackOffice.Api.json -o shared/lib/api/api.generated.d.ts",
     "publish": "npm run build && mkdir -p ../Api/publish/WebApp/dist/ && rsync -r ./dist/ ../Api/publish/WebApp/dist/"
   },
-  "dependencies": {
-    "react-aria-components": "^1.2.1",
-    "react-dom-confetti": "^0.2.0"
-  },
   "devDependencies": {
     "@repo/build": "*",
     "@repo/config": "*",
     "@repo/infrastructure": "*",
     "@repo/ui": "*"
-  },
-  "postcss": {
-    "plugins": {
-      "tailwindcss": {}
-    }
-  },
-  "browserslist": ["last 1 version", "> 1%", "not dead"]
```

Comment:
required / recommended

Reply:
I’ve tested this using npm install browserlist. After moving this to the root package.json, running browser list for one self-contained system respects the setting in the root. So this will both simplify the package.json for each self-contained system and give us operational consistency.